src/remote.go

```diff
@@ -649,7 +649,9 @@ func (r *Remote) findByPathRecvRaw(parentId string, p []string, trashed bool) (f
 	// TODO: use field selectors
 	var expr string
 	head := urlToPath(p[0], false)
-	quote := strconv.Quote
+	quote := func(s string) string {
```

Comment:
Could you instead add this function to `src/misc.go` can call it something like `customQuote` and then inside this file `src/remote.go` search and replace `quote` with `customQuote`?